projects/media-viewer/src/lib/toolbar/search-bar/search-bar.component.html

```diff
@@ -1,39 +1,60 @@
-<div class="findbar doorHanger" [hidden]="toolbarEvents.searchBarHidden | async">
-  <div id="findbarInputContainer">
-    <input #findInput
-           [(ngModel)]="searchText"
-           (ngModelChange)="search()"
-           (keydown)="onInputKeyPress($event)"
-           [ngClass]="'toolbarField ' + (!searchText || haveResults ? '' : 'notFound')"
-           title="Find"
-           placeholder="Find in document…"
-           tabindex="0"
-           data-l10n-id="find_input">
-    <div class="splitToolbarButton">
-      <button id="findPrevious" class="toolbarButton findPrevious"
-              title="Find the previous occurrence of the phrase" tabindex="0" data-l10n-id="find_previous"
-              (click)="searchPrev()">
-        <span data-l10n-id="find_previous_label">Previous</span>
-      </button>
-      <div class="splitToolbarButtonSeparator"></div>
-      <button id="findNext" class="toolbarButton findNext" title="Find the next occurrence of the phrase"
-              tabindex="0" data-l10n-id="find_next" (click)="searchNext()">
-        <span data-l10n-id="find_next_label">Next</span>
+<div class="searchbar govuk-!-padding-3" [hidden]="toolbarEvents.searchBarHidden | async">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      <input class="govuk-input govuk-!-display-inline-block govuk-!-width-three-quarters govuk-!-margin-bottom-1"
+        type="text" #findInput [(ngModel)]="searchText" (keydown)="onInputKeyPress($event)" title="Find"
```

Comment:
can we filter on the keys we want here instead of listening for all keys, so we would just do (keydown.space), (keydown.escape), (keydown.enter), with a corresponding method for each

Reply:
I've updated the code and this will be available in the next commit.